lab-integration/src/main.ts

```diff
@@ -0,0 +1,49 @@
+import { promises as fs, watch } from "fs";
+import path from "path";
+import { TestsResult } from "./types";
+
+const filePath = path.join(process.cwd(), "../test-results.json");
+
+(async () => {
+  console.info("SCRIPT IS RUNNING...");
+
+  const stringifiedDataset = await fs.readFile(filePath, "utf8");
+  const dataset = JSON.parse(stringifiedDataset) as TestsResult[];
+  let lastTestResultDate: Date = getLastTestsResulstDate(dataset);
+
+  watch(filePath, (eventType) => {
+    if (eventType === "change") {
+      const newTestsResults = dataset.filter(
+        (testResult) => convertToDate(testResult.Data) > lastTestResultDate,
+      );
+      lastTestResultDate = getLastTestsResulstDate(dataset);
```

Comment:
It's a minor, but we can pass `newTestsResults` to `getLastTestsResulstDate`, right?